common/Scripts/ModifyWindowsOptionalFeatures.cs

```diff
@@ -13,105 +13,121 @@
 
```

Comment:
Do the contents of the Scripts folder make more sense in the Customization project?

Reply:
it could. originally the idea was that environments/hyper-v and other components could leverage this same script instead of duplicating the logic. That didn't quite pan out in the original change and hasn't been revisited. I think for now, I'd prefer to keep it in common, since long paths is also there, and we'd want to move both as a separate individual change.